extension/content.js

```diff
@@ -268,7 +268,11 @@ function linkifyIssuesInTitles() {
 }
 
 function addPatchDiffLinks() {
-	const commitUrl = location.href.replace(/\/$/, '');
+	let commitUrl = location.href.replace(/\/$/, '');
+	const prCommit = /\/pull\/\d+\/commits/;
+	if (prCommit.test(commitUrl)) {
+		commitUrl = commitUrl.replace(prCommit, '/commit');
```

Comment:
I figured it wouldn't be this easy :) Do you have an example link to a page where it breaks down?